src/com/oltpbenchmark/benchmarks/tpcc/ddls/tpcc-postgres-ddl.sql

```diff
@@ -56,7 +56,7 @@ CREATE TABLE oorder (
   o_ol_cnt decimal(2,0) NOT NULL,
   o_all_local decimal(1,0) NOT NULL,
   o_entry_d timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
-  PRIMARY KEY (o_w_id,o_d_id,o_id),
+  PRIMARY KEY ((o_w_id,o_d_id) hash,o_id),
   UNIQUE (o_w_id,o_d_id,o_c_id,o_id)
```

Comment:
Please remove this and add a `CREATE UNIQUE INDEX` statement below using `((o_w_id, o_d_id) HASH, o_c_id, o_id)`